model_fitting/module_wraper.py

```diff
@@ -22,7 +22,7 @@ def repeat_items(list):
                       
 def build_classifier(first_phase_output_path, motif_inference_output_path,
                      classification_output_path, logs_dir, samplename2biologicalcondition_path, number_of_random_pssms,
-                     fitting_done_path, check_files_valid, stop_before_random_forest, num_of_random_configurations_to_sample,
+                     fitting_done_path, check_files_valid, stop_before_random_forest, run_local_randon_forst_for_bc, num_of_random_configurations_to_sample,

```

Comment:
`randon => random`.
Please recheck and validate names, use a spellchecker if needed.

No indication that this variable is a boolean, please add `is_` prefix.

Also, the name is inaccurate, setting it to false would still run the configurations in parallel.
Change to `is_run_random_forest_per_bc_sequentially` and in the help explain what is parallel and when to set this to True